app/components/cases/subcase-mandatees/component.js

```diff
@@ -1,22 +1,21 @@
 import Component from '@ember/component';
 import { inject } from '@ember/service';
 import EmberObject from '@ember/object';
-import ManageMinisterMixin from 'fe-redpencil/mixins/manage-minister-mixin';
 
-export default Component.extend(ManageMinisterMixin, {
+export default Component.extend({
   store: inject(),
   classNames: ['vlc-input-field-block'],
   isAdding: false,
   isEditingMandateeRow: false,
 
-  getDomainOfField(field) {
+  async getDomainOfField(field) {
     if (field)
-      return field.get('domain');
+      return await field.get('domain');
   },
 
-  getFieldOfIseCode(iseCode) {
+  async getFieldOfIseCode(iseCode) {
     if (iseCode)
-      return iseCode.get('field');
+      return await iseCode.get('field');
```

Comment:
Why did this change to async await in the context of a mixin refactor?

Reply:
Confusion about the use of async or not when getting properties.

Reply:
Does it work without it? I wouldn't change this otherwise

Reply:
Ok